src/routes/_surfaces/workspace/Onboarding.svelte

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+  import ParquetIcon from "$lib/components/icons/Parquet.svelte";
+  import Code from "$lib/components/icons/Code.svelte";
+  // import Database from "$lib/components/icons/Database.svelte";
+  // import SurfaceView from "$lib/components/icons/SurfaceView.svelte";
+
+  const steps = [
+    {
+      heading: "import your data source",
+      description:
+        "Add to your sources by dragging a csv or parquet file to this window.",
```

Comment:
Can we also mention you can hit the "+" icon in the Sources list?

Reply:
@magorlick sent me this copy. @magorlick should we also mention about the "+" icon?

Reply:
You can add "or clicking '+' under SOURCES" to the first bullet's text.

Reply:
Just got the green light from her, so I just made the edit